packages/react-examples/src/react-charting/SankeyChart/SankeyChart.Rebalance.Example.tsx

```diff
@@ -0,0 +1,95 @@
+import { IChartProps, ISankeyChartProps, SankeyChart } from '@fluentui/react-charting';
+import * as React from 'react';
+
+interface ISankeyChartBasicState {
+  width: number;
+  height: number;
+}
+
+export class SankeyChartRebalanceExample extends React.Component<{}, ISankeyChartBasicState> {
```

Comment:
Could you add this example to https://github.com/microsoft/fluentui/blob/master/packages/react-examples/src/react-charting/SankeyChart/SankeyChartPage.tsx so that it shows up on the demo site.

Reply:
Done.